### PR TITLE
ui img

### PR DESCRIPTION
**Pull request by yasintazeoglu**

### CODE REVIEW

File: src/components/pages/communities.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class CommunityList extends Component {
         <div style={style.createCommunities}>
           <div style={style.text}>{"Can't find what you are looking for?"}</div>
           <span style={style.button}>
-            <Button title="Create Your Community" onClick={() => {}} size="xlarge" type="secondary" style={{ alignItems: 'center' }}/>
+            <Button label="Create Your Community" onClick={() => {}} size="large" type="secondary" style={{ alignItems: 'center' }}/>
```
Comment (SelmanKahya) [inline src/components/pages/communities.js]: why did you change it to large?

File: src/components/ui/img.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+import LazyLoad from 'react-lazyload';
+
+const style = {};
```
Comment (SelmanKahya) [inline src/components/ui/img.js]: you can remove this

File: src/components/ui/img.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+const UIImage = (props) => {
+  const {
+    src, alt, lazy, height,
+  } = props;
```
Comment (SelmanKahya) [inline src/components/ui/img.js]: it should take className as a prop and apply to the image

File: src/components/pages/communities.js (hunk 1 above)
Comment (yasintazeoglu) [reply src/components/pages/communities.js]:
```
UIButton.propTypes = {
  disabled: PropTypes.bool,
  extraClassName: PropTypes.string,
  onClick: PropTypes.func,
  size: PropTypes.oneOf(['small', 'medium', 'large']).isRequired,
  label: PropTypes.string.isRequired,
  type: PropTypes.oneOf(['primary', 'secondary', 'outline', 'danger', 'plain']).isRequired,
};
```
src/components/ui/button.js propTypes
